### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by sharmine-s**
- Created Navbar component as a child component of Feed
- Moved logout button and functionality into new Navbar component
- Navbar only shows when user is logged in
- Added CSS styling to make it look nice

### CODE REVIEW

File: api/spec/controllers/tokens.spec.js
Comment (michael-szczepanski) [inline api/spec/controllers/tokens.spec.js:1]: Is there a particular reason why these tests got greyed out?

**Review by michael-szczepanski — CHANGES_REQUESTED** [review]
The commented out tests in backend tokens spec seem to pass on my machine. Was this just forgotten about?

**Comment by sharmine-s** [comment]
@michael-szczepanski oops that was a mistake! good catch, resubmitted

**Review by wimdavies — APPROVED** [review]
Nice work!

**Review by wimdavies — APPROVED** [review]
lgtm

**Review by michael-szczepanski — APPROVED** [review]

**Pull request merged**